html/camera-positioning.html

```diff
@@ -5,19 +5,26 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="/static/css/bootstrap.min.css">
+    <script type="text/javascript" src="/static/js/jquery-3.3.1.slim.min.js"></script>
```

Comment:
This is already pulled in further down in the template.

According to the bootstrap docs the jquery and bootstrap JS files should be included right at the bottom of the HTML. I presume this lets the browser start downloading all assets for the page without blocking on JS parsing.